gpt4all-backend/llmodel.h

```diff
@@ -66,6 +66,7 @@ class LLModel {
         int32_t n_predict = 200;
         int32_t top_k = 40;
         float   top_p = 0.9f;
+        float   min_p = 0.1f;
```

Comment:
Should set the default to zero to match the other places:
```suggestion
        float   min_p = 0.0f;
```

Reply:
Fixed as well.